resources/lambda/browsershot.js

```diff
@@ -27,6 +27,24 @@ exports.handle = async function (event) {
     // HTML in a temporary file and use the path as the URL.
     if (event._html) {
         fs.writeFileSync('/tmp/index.html', event._html);
+        event.url = 'file:///tmp/index.html';
+    } else if (event.options.s3Source) {
+        // If the source is S3, then download the file into a temporary file to be used as the URL.
+        const s3 = new AWS.S3({
+            region: event.options.s3Source.region,
+            accessKeyId: event.options.s3Source.key,
+            secretAccessKey: event.options.s3Source.secret,
```

Comment:
`key` and `secret` are not set in `BrowsershotLambda@htmlFromS3File()` … but I've only know realised that it's also not set in the code related to `saveToS3()`.

As my local tests are also green, I would merge this as is.
Have to investigate and document, what these values actually are and if they are set automatically.